MetadataExtractor/Formats/QuickTime/QuickTimeMetadataHeaderDescriptor.cs

```diff
@@ -14,6 +14,8 @@ public QuickTimeMetadataHeaderDescriptor(QuickTimeMetadataHeaderDirectory direct
             return tagType switch
             {
                 QuickTimeMetadataHeaderDirectory.TagArtwork => GetArtworkDescription(),
+                QuickTimeMetadataHeaderDirectory.TagCreationDate => Directory.GetDateTime(tagType).ToString("yyyy:MM:dd HH:mm:ss"),
+                QuickTimeMetadataHeaderDirectory.TagLocationDate => Directory.GetDateTime(tagType).ToString("yyyy:MM:dd HH:mm:ss"),
```

Comment:
I think it's better to store `DateTime` in the `QuickTimeMetadataHeaderDirectory` rather than string. This allows the default logic to apply locale-specific formatting for dates.